src/libYARP_math/include/yarp/math/FrameTransform.h

```diff
@@ -13,121 +13,70 @@
 #include <yarp/math/Math.h>
 #include <yarp/math/Quaternion.h>
 
+#define VALID_POSITION         1  // 0b00000001
+#define VALID_ROTATION         2  // 0b00000010
+#define VALID_LIN_VELOCITY     4  // 0b00000100
```

Comment:
Ususally instead of write directly the numbers `2`, `4` and so on, those values are written like `1 << 1`, `1 << 2` in such a way it is more clear they are used for bitwise operation and not as a value.